src/content/learn/editor-setup.md

```diff
@@ -1,62 +1,62 @@
 ---
-title: Editor Setup
+title: Cài đặt trình soạn thảo văn bản
 ---
 
 <Intro>
 
-A properly configured editor can make code clearer to read and faster to write. It can even help you catch bugs as you write them! If this is your first time setting up an editor or you're looking to tune up your current editor, we have a few recommendations.
+Một trình soạn thảo văn bản được cấu hình đúng cách có thể làm cho mã nguồn dễ đọc hơn và tốc độ viết code nhanh hơn. Nó còn có thể giúp bạn phát hiện lỗi khi bạn đang viết code! Nếu đây là lần đầu tiên bạn cài đặt một trình soạn thảo văn bản hoặc bạn đang muốn tinh chỉnh trình soạn thảo văn bản hiện tại của mình, chúng tôi có vài đề xuất.
 
 </Intro>
 
-<YouWillLearn>
+<BạnSẽBiết>
 
-* What the most popular editors are
-* How to format your code automatically
+* Các trình soạn thảo phổ biến nhất
+* Làm sao để định dạng code một cách tự động
 
-</YouWillLearn>
+</BạnSẽBiết>
```

Comment:
This is a bit weird to show as a jsx tag. I don't think it's a good idea to translate it.

Reply:
Thanks, I will revert it